scripts/main/lychee.js

```diff
@@ -789,17 +971,36 @@ lychee.loadDropbox = function (callback) {
 	} else if (lychee.dropbox === true && lychee.dropboxKey != null && lychee.dropboxKey !== "") {
 		callback();
 	} else {
-		settings.setDropboxKey(callback);
+		// TODO: Is this branch ever called?
+		// In particular, this branch behaves differently from the other two
+		// in the sense that it neither loads the Dropbox component nor
+		// calls the callback.
+		settings.setDropboxKey();
```

Comment:
pretty sure this one is legacy from Tobias.

Reply:
Completely revised the method.